library/old_protocol/translate_request.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <atomic>
+
+#include "library/n2_protocol.hpp"
+#include "serialize.hpp"
+
+namespace ioremap { namespace elliptics { namespace n2 {
+
+class replier_base {
```

Comment:
I think I've seen pretty same struct with dynamicly dispatched std::functions

Reply:
Yes, it is backend for these std::functions